### PR TITLE
chore(RecoveryPhrase): Add UI for removing recovery seed from the account

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖
Adds the UI for removing/deleting the mnemonic phrase from the account.

### CODE REVIEW

File: common/locales/pl/pl.ftl [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+    .join = Dołącz 
+    .joined = Dołączono!
+
+updates = Aktualizacje 
+    .title = Aktualizacja Uplink jest dostępna. Postępuj według danych kroków i zainstaluj najnowszą aktualizację. 
+    .instruction1 = 1. Pobierz aktualizację z naszej strony 
+    .instruction2 = 2. Zamknij Uplink 
+    .instruction3 = 3. Otwórz instalator. Starsza wersja Uplink będzie nadpisana, pozostawiając twoje konto nienaruszone. 
+    .instruction4 = 4. Otwórz Uplink i rozmawiaj ponownie! 
+    .instruction5 = *Usprawnimy ten proces w następnych aktualizacjach
```
[the remark this answers is not in the document]
Comment (InfamousVague) [reply common/locales/pl/pl.ftl:60]: Wrong

File: common/locales/pl/pl.ftl [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+settings-keybinds = Ustawienia klawiszy
+    .increase-font-size = Powiększ rozmiar czcionki w aplikacji Uplink
+    .decrease-font-size = Zmniejsz rozmiar czcionki w aplikacji Uplink
+    .change-keybind = Skrót klawiszowy dla rozpoczęcia nagrywania
+    .cancel-change-keybind = Skrót klawiszowy dla anulowania nagrywania
+
+settings-extensions = Ustawienia rozszerzeń 
+    .open-extensions-folder = Otwórz Rozszerzenia
+    .auto-enable = Uruchom automatycznie
+    .auto-enable-description = Po włączeniu, nowe rozszerzenia będą uruchamiane automatycznie
```
[the remark this answers is not in the document]
Comment (InfamousVague) [reply common/locales/pl/pl.ftl:289]: Wrong

**Comment by InfamousVague** [comment]
image
image
image

**Comment by phillsatellite** [comment]
@InfamousVague Just wanted to run this by you to see if this is intended 

-On a new account, I went to Settings and removed my recovery seed from my account 
-I closed out the app and re opened 
-When I then went to Settings again my recovery phrase was back in Settings 

https://github.com/Satellite-im/Uplink/assets/93608357/516fcf8b-041e-4325-9470-70be9b305eae

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**